modules/claims_api/app/controllers/claims_api/v0/forms/intent_to_file_controller.rb

```diff
@@ -36,7 +36,7 @@ def check_future_type
               errors: [
                 {
                   status: 422,
-                  details: "#{form_type.titelize} claims are not currently supported, but will be in a future version"
+                  details: "#{form_type.titleize} claims are not currently supported, but will be in a future version"
```

Comment:
Can you include these in a quick separate PR that we can merge in right away since this is technically broken in prod at the moment? Thanks